src/Microsoft.Graph/Generated/callrecords/requests/CallRecordRequest.cs

```diff
@@ -60,6 +60,29 @@ public async System.Threading.Tasks.Task<CallRecord> CreateAsync(CallRecord call
             return newEntity;
         }
 
+        /// <summary>
+        /// Creates the specified CallRecord using POST and returns a <see cref="GraphResponse{CallRecord}"/> object.
+        /// </summary>
+        /// <param name="callRecordToCreate">The CallRecord to create.</param>
+        /// <returns>The <see cref="GraphResponse{CallRecord}"/> object of the request.</returns>
+        public System.Threading.Tasks.Task<GraphResponse<CallRecord>> CreateResponseAsync(CallRecord callRecordToCreate)
```

Comment:
Would it be possible to only have one overload instead with an optional parameter? (i.e. set the cancellationToken parameter value to None)

Reply:
Unfortunately, this is a bit tricky, as CancellationToken.None is not a compile time constant, which is a requirement for default values in optional arguments.
I believe that's why the other functions follow the same pattern.

Reply:
what about setting the default value to null then and if it's null, then we assign none?

Reply:
(we should probably do that in the core SDK as a defensive programming mechanism instead of the generated part)

Reply:
Looks like setting to null also won't work. 
The following error is raised.
`A value of type '<null>' cannot be used as a default parameter because there are no standard conversions to type 'CancellationToken'`

We could actually probably do 
`CancellationToken cancellationToken = default` and then add the check in the core ibrary as you suggested.

Reply:
oh yeah, I had forgotten about the niceness of default, spending too much time in java land...

Reply:
Looks like according to [this](https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.none?redirectedfrom=MSDN&view=net-5.0#remarks), the will be no need for the core library check since `default(CancellationToken)` is the same as `CancellationToken.None`